comps/guardrails/pii_detection/predictionguard/README.md

```diff
@@ -0,0 +1,48 @@
+# PII Detection Prediction Guard Microservice
+
+[Prediction Guard](https://docs.predictionguard.com) Prediction Guard allows you to seamlessly integrate private, controlled, and compliant Large Language Models (LLM) functionality. In addition to providing a scalable LLM API, we enable you to prevent hallucinations, institute governance, and ensure compliance. Using Prediction Guard gives you quick and easy access to state-of-the-art LLMs. Acquire an API key by going [here](https://mailchi.mp/predictionguard/getting-started).
+
+Detecting Personal Identifiable Information (PII) is important in ensuring that users aren't sending out private data to LLMs.
+
+# 🚀1. Start Microservice with Python
+
+## 1.1 Install Requirements
+
+```bash
+pip install -r requirements.txt
+```
+
+# 🚀2. Start Microservice with Docker
+
+## 2.1 Setup Environment Variables
+
+Setup the following environment variables first
+
+```bash
+export PREDICTIONGUARD_API_KEY=${your_predictionguard_api_key}
+```
+
+## 2.1 Build Docker Images
+
+```bash
+cd ../..
+docker build -t opea/pii_predictionguard:latest -f comps/guardrails/pii_detection/predictionguard/docker/Dockerfile .
```

Comment:
Similar comments on PG blurb and dashes vs underscores.

Reply:
Fixed.